### ISSUE TITLE
vulnerability found with jquery ^3.3.1

### ISSUE THREAD

**Issue by sukrut-gs** [TriplyDB/Yasgui#158]
vulnerability found with jquery ^3.3.1. Kindly upgrade to 3.5.0.
The report says

Regex in its jQuery.htmlPrefilter sometimes may introduce XSS.

**Comment by sukrut-gs** [comment]
https://snyk.io/test/npm/jquery/3.3.1

**Comment by sukrut-gs** [comment]
@GerwinBosch  when you are releasing this change..?

**Comment by GerwinBosch** [comment]
@sukrut-gs Sorry, give me 5 minutes

**Comment by sukrut-gs** [comment]
ok. cool

**Comment by GerwinBosch** [comment]
New version should be available now

**Comment by sukrut-gs** [comment]
yeah.. thanks.

**Comment by sukrut-gs** [comment]
Hey still getting

node_modules/@triply/yasr/build/yasr.min.js
--
Regex in its jQuery.htmlPrefilter sometimes may introduce XSS

**Comment by sukrut-gs** [comment]
@GerwinBosch  in yasgui.min.js and yasr.min.js it is still using 3.4.1 I think
S.extend({expando:"jQuery"+("3.4.1"+Math.random()).replace(/\D/g,"")

**Comment by GerwinBosch** [comment]
@sukrut-gs The library used for Yasr's table plugin is using an older version of jquery.  replacing the library used for the table plugin would take a lot more effort than updating the `package.json`

**Comment by GerwinBosch** [comment]
It seems that some of the visualization plugins use an older version of jquery, run `yarn why jquery`.

Could you check if upgrading these packages would resolve the issue?

**Comment by sukrut-gs** [comment]
I got this
=> Found "datatables.net-dt#jquery@3.4.1"
info This module exists because "@triply#yasgui#@triply#yasr#datatables.net-dt" depends on it.
=> Found "datatables.net#jquery@3.4.1"
info This module exists because "@triply#yasgui#@triply#yasr#datatables.net" depends on it.
=> Found "pivottable#jquery@3.4.1"
info This module exists because "@triply#yasgui#@triply#yasr#pivottable" depends on it.

**Comment by sukrut-gs** [comment]
Data tables have released newer version 2 days back.
https://github.com/DataTables/Dist-DataTables-DataTables/blob/master/package.json
But they are still using "jquery": ">=1.7"
@GerwinBosch

**Comment by GerwinBosch** [comment]
Yep, that is similar to what I got, does updating those three packages resolve the issue?

**Comment by sukrut-gs** [comment]
I do not think.. because if you see datatables still uses jQuery >=1.7

**Comment by GerwinBosch** [comment]
In that case I'd look for a different plugin, however that's a lot more work then just updating some packages. @LaurensRietveld do you have any input?
